types/fg-loadcss/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for fg-loadcss 3.1
+// Project: https://github.com/filamentgroup/loadCSS
+// Definitions by: Noah Overcash <https://github.com/ncovercash>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace loadCSS;
+
+declare function loadCSS(
+    href: string,
+    before?: HTMLElement,
+    media?: string,
+    attributes?: Record<string, string>,
+): HTMLLinkElement;
+
+export = loadCSS;
```

Comment:
```suggestion
export function loadCSS(
    href: string,
    before?: HTMLElement,
    media?: string,
    attributes?: Record<string, string>,
): HTMLLinkElement;
```
you're correct, this works. There is only single, named export in this module